spec/models/app_feature_completion_spec.rb

```diff
@@ -2,39 +2,32 @@
 
 RSpec.describe AppFeatureCompletion do
   describe ".satisfied_by" do
-    let(:split) { FactoryBot.create(:split) }
     let(:app) { FactoryBot.create(:app) }
     let(:app_build) { app.define_build(built_at: Time.zone.now, version: "1.0") }
     let(:different_app) { FactoryBot.create(:app) }
 
-    it "blows up if you don't BYO splits table" do
-      FactoryBot.create(:app_feature_completion, app: app, split: split, version: "1.0")
-
-      expect { described_class.satisfied_by(app_build).to_a }.to raise_error(/missing FROM-clause entry for table "splits"/)
-    end
```

Comment:
Note to self: don't test implementation details :) That said, I was pretty sure this was gonna have to work like this forever when I wrote the test and wanted to make sure I didn't accidentally regress the split scoping or else it would've returned incorrect SQL all over the place. Instead now, we test the split join behavior in split_spec.